tasks/prime.py

```diff
@@ -215,6 +216,65 @@ class PrimeTasks(PrimeDataStorageMixin, ParserTaskMixin, CertTaskMixin, TaskSet)
     tags where appropriate to make filtering for custom tests easier.
     """
 
+    def __init__(self, parent):
+        self.csrf_token = None
+        self.session_token = None
+        super().__init__(parent)
+
+    def customer_path(self, url: str) -> str:
+        return f"{self.user.alternative_host}{url}"
+
+    def on_start(self):
```

Comment:
@pearl-truss It's likely that the entirety of the on_start function should be refactored. The first part of the code is just setting up the cookies so that we can login while the latter portion is going through the APIs to do the profile setup. The cookie setup functionality already exist in an abstracted form in the `LoginTaskSet` class. I'm hoping that once we have a crude version of this working and the tests are fixed, we will more context to finalize how do we structure the code.

Reply:
This runs for every instance of the prime user too right?  So if we generate 100 prime users it would initiate 100 new customers?

Reply:
Yeah, that is correct. There are ways to run a function per test rather than per user, but we don't have access to the client that makes the API calls there (at least by default).